Assets/Editor/SchedulerEditorTest.cs

```diff
@@ -0,0 +1,311 @@
+﻿#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Text;
+using System.Xml;
+using NUnit.Framework;
+using Scheduler;
+
+public class SchedulerEditorTest
+{
+    private Scheduler.Scheduler scheduler;
+    private Action<ScheduledEvent> callback;
+
+    [SetUp]
+    public void Init()
+    {
+        if (PrototypeManager.SchedulerEvent == null)
+        {
+            new PrototypeManager();
```

Comment:
This is very very very bad idea to have tests linked directly to file system. ( `FurniturePrototypes` )